setup.py

```diff
@@ -101,7 +101,7 @@
       packages=find_packages(exclude=["*.tests", "*.tests.*",
                                       "tests.*", "tests"]),
       install_requires=[
-          'torch>=0.4.1,<1.2',
+          'torch>=1.2.0',
```

Comment:
Do we strictly need features from 1.2.0 or is 1.0 sufficient?

Reply:
this PR is in response to breaking changes that happened between 1.1 and 1.2, but also going forward we'd like to use PyTorch's dataloader infrastructure, and for that to support allennlp datasets we need torch>=1.2

Reply:
Got it. Thanks for explaining.